### PR TITLE
Update migration docs

### PR DESCRIPTION
**Pull request by shubham-mahajan**
The changes made are to make the migration process as smooth as possible and fix the potential issues that may arise.

### CODE REVIEW

File: dx/data-migration/README.md [hunk 1]
```diff
@@ -25,6 +27,15 @@ This article is still under testing and development.
     # Copy something:
     $ mc cp -r prod/ckan/montreal-dev/uploads dx/dx-montreal-staging/
     ```
+6. Update the permission of `uploads` so that they are publically accessible and the images for groups/organizations can be loaded
+    ```
+    gsutil acl ch -r -u AllUsers:R gs://<bucket-name>/<file-location>/<file>
```
Comment (Irio) [inline dx/data-migration/README.md]: How do you apply that for all the files recursively? I assume you didn't do that once for every different file.

**Comment by anuveyatsu** [comment]
@shubham-mahajan do we still need this PR? Can you check vs #8 ?

**Comment by shubham-mahajan** [comment]
> @shubham-mahajan do we still need this PR? Can you check vs #8 ?

@anuveyatsu I need to update the docs little bit now.

File: dx/data-migration/README.md (hunk 1 above)
Comment (shubham-mahajan) [reply dx/data-migration/README.md]: Updated the docs

**Comment by shubham-mahajan** [comment]
> > @shubham-mahajan do we still need this PR? Can you check vs #8 ?
> 
> @anuveyatsu I need to update the docs little bit now.

@anuveyatsu Fixed

**Pull request merged**